vendor/src/github.com/docker/engine-api/types/types.go

```diff
@@ -274,6 +274,27 @@ type ExecStartCheck struct {
 	Tty bool
 }
 
+// HealthcheckResult stores information about a single run of a healthcheck probe
+type HealthcheckResult struct {
+	Start, End time.Time
+	ExitCode   int    // 0=healthy, 1=unhealthy, 2=starting, -1=error running probe
+	Output     string // Output from last check
```

Comment:
Look, without a healthcheck protocol, assuming anything about the output of a healthcheck is silly. What if there are several aspects to health check that need to be displayed? What if someone wants to use a binary protocol for the healthcheck? What about debugging?

The salient portion of the healthcheck my not even be in the first 255 bytes.

I'm sorry, but this is a ridiculous limitation.